doc/alignment.md

```diff
@@ -0,0 +1,61 @@
+
+Partition Alignment
+===================
+
+Unfortunately these days for optimal performance care must be taken that
+partitions are properly aligned, see e.g.
+https://en.wikipedia.org/wiki/Partition_alignment.
```

Comment:
Thanks. Then my personal test says that I'd expect mentioning at least a tiny bit of this paragraph and link to Advanced_Format